docs/OMF.rst

```diff
@@ -459,12 +459,29 @@ Asset Framework Location Hint
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 An Asset Framework location hint can be added to a reading to control
-the placement of that asset within the Asset Framework. An Asset Framework
+the placement of the asset within the Asset Framework. An Asset Framework
 hint would be as follows:
 
 .. code-block:: console
 
    "OMFHint"  : { "AFLocation" : "/UK/London/TowerHill/Floor4" }
+   
+Note the following when defining an *AFLocation* hint:
+
+- An asset in a Fledge Reading is used to create a `Container in the OSIsoft Asset Framework <https://docs.osisoft.com/bundle/omf-with-pi-web-api/page/container-messages.html>`_.
+  A *Container* is an AF Element with an AF Attribute that is mapped to a PI Point using the OSIsoft PI Point Data Reference.
+  The names of both the AF Element and the AF Attribute are the Fledge Reading asset name.
+- If you edit the AF Location hint, the Container will be moved to the new location in the AF hierarchy.
+- If you disable the OMF Hint filter, the Container will not move.
+- If you wish to move a Container, you can do this with the PI System Explorer.
+  Right-click on the AF Element that represents the Container.
+  Choose Copy.
+  Select the AF Element that will serve as the new parent of the Container.
```

Comment:
Would be better if we attach screenshots?

Reply:
I am trying to avoid describing the PI points created by OMF and stick with the AF Element and AF Attributes that are mapped to them. You raise a good point about a Fledge asset with more than one data point. I presume these become more than one AF Attribute within an AF Element. My main point is that an administrator can move the AF Element; we don't have to do it for them. Regarding screenshots, it might help but it would also make that section of documentation large and hard to navigate. We would also be documenting AVEVA's product.